react/Image.tsx

```diff
@@ -117,6 +119,9 @@ function Image(props: ImageProps) {
       style={imageDimensions}
       ref={imageRef}
       className={handles.imageElement}
+      {...(preload ? {
+        'data-vtex-preload': 'true'
+      } : {})}
```

Comment:
can't it be just false?

Reply:
not sure actually, I don't remember what happens when I try to destructure "false", left it this way to be more safe